### PR TITLE
Add Sign up Page and functionality to save new user data at fake db

### PR DESCRIPTION
**Pull request by Johannafendrich**
image

#5

### CODE REVIEW

File: client/src/pages/SignUp.js [hunk 1]
```diff
@@ -1,6 +1,87 @@
...
+            onChange={(event) => {
+              setUserPassword(event.target.value);
+            }}
+          />
+
+          <Container>
+            <Button>Create account</Button>
+          </Container>
+          <Supplement>
+            {'Already have an account?'}
```
Comment (lmachens) [inline client/src/pages/SignUp.js:79]:
```suggestion
            Already have an account?
```

Comment (lmachens) [reply client/src/pages/SignUp.js:79]: or
```suggestion
            <span>Already have an account?</span>
```

File: client/src/pages/SignUp.js [hunk 2]
```diff
@@ -1,6 +1,87 @@
...
+            <Link to="/">{'Login here'}</Link>
```
Comment (lmachens) [inline client/src/pages/SignUp.js:80]:
```suggestion
            <Link to="/">Login here</Link>
```

File: client/src/pages/SignUp.js (hunk 1 above)
Comment (Johannafendrich) [reply client/src/pages/SignUp.js]: Thanks for your suggestion :) I put a span around the text 👍🏼

**Pull request merged**